### PR TITLE
Add zsh features, remove dock updates, and update README

### PR DESCRIPTION
**Pull request by drewdrewthis**
- Adds syntax highlighting and Z command to zsh
- Removes dock updates (no longer works with latest OSXs
- Updates README for capybara/Sierra issues with Xcode

### CODE REVIEW

**Comment by drewdrewthis** [comment]
@policygenius/developers CR Plz?

File: README.md [hunk 1]
```diff
@@ -93,6 +123,8 @@ What it sets up
 [Tmux]: http://tmux.sourceforge.net/
 [Zsh]: http://www.zsh.org/
 [Oh My ZSH!]: http://ohmyz.sh/
+[ZSH Syntax Highlighting]: https://github.com/zsh-users/zsh-syntax-highlighting
+[Z]: https://github.com/rupa/z
```
Comment (ianyamey) [inline README.md:127]: I think most in the team have a preference for using `j` instead of `z`.

File: README.md [hunk 2]
```diff
@@ -30,6 +32,32 @@ less mac.sh
...
+
+   `vim /usr/local/Cellar/qt@5.5/5.5.1/mkspecs/features/mac/default_pre.prf`
+
+3. Find the line with this text (for me it was line 15):
+
+   `isEmpty($$list($$system("/usr/bin/xcrun -find xcrun 2>/dev/null"))): \`
+
+4. Replace line with:
+
+   `isEmpty($$list($$system("/usr/bin/xcrun -find xcodebuild 2>/dev/null"))): \`
```
Comment (bradwheel) [inline README.md:54]: 👍 for documenting this.

File: README.md (hunk 1 above)
Comment (drewdrewthis) [reply README.md:127]: `z` is pretty sweet... but `j` is not part of the laptop script anyway. I don't think there's any reason they can't both be added, but one doesn't preclude the other.

**Review by ianyamey — APPROVED** [review]

**Pull request merged**